kratos/utilities/mortar_utilities.cpp

```diff
@@ -163,22 +164,20 @@ void ComputeNodesTangentModelPart(
 
     // We iterate over nodes
     auto& r_nodes_array = rModelPart.Nodes();
-    const auto it_node_begin = r_nodes_array.begin();
-    const int num_nodes = static_cast<int>(r_nodes_array.size());
 
-    #pragma omp parallel for
-    for(int i = 0; i < num_nodes; ++i) {
-        auto it_node = it_node_begin + i;
-
-        // Computing only slave nodes
-        if (it_node->Is(SLAVE)) {
-            if (has_lm && !SlipAlways) {
-                ComputeTangentNodeWithLMAndSlip(*it_node, 0, pSlipVariable, SlipCoefficient, domain_size);
-            } else {
-                ComputeTangentNodeWithSlip(*it_node, 0, pSlipVariable, SlipCoefficient, domain_size);
+    block_for_each(
+        r_nodes_array,
+        [&pSlipVariable,SlipCoefficient,domain_size,has_lm,SlipAlways](Node<3>& r_node)
```

Comment:
```suggestion
        [&pSlipVariable,SlipCoefficient,domain_size,has_lm,SlipAlways](Node<3>& rNode)
```

@matekelemen I know this is a very minor request, but could you please make this PR and the others style guide conforming?
Thx!